apps/cloud_server.c

```diff
@@ -1535,6 +1535,21 @@ cloud_server_log(oc_log_level_t log_level, oc_log_component_t component,
   fflush(stdout);
 }
 
+static void
+cloud_provision_conf_resource(oc_cloud_context_t *ctx)
+{
+  if (cis == NULL || auth_code == NULL || sid == NULL) {
+    return;
+  }
+  oc_cloud_provision_conf_resource(ctx, cis, auth_code, sid, apn);
+
+  oc_uuid_t id = {
+    { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 15, 16 },
+  };
+  oc_cloud_add_server(ctx, "coap+tcp://mock.cloud:5683",
+                      CHAR_ARRAY_LEN("coap+tcp://mock.cloud:5683"), id);
```

Comment:
why this is added here ? - it only works for unsecure device.

Reply:
ah sorry, I was testing the GET output for device with multiple servers, needs to be removed

Reply:
removed